src/tests/Microsoft.PowerFx.Core.Tests.Shared/TexlTests.cs

```diff
@@ -4203,13 +4204,16 @@ public void TestSilentValidDelegatableFilterPredicateNode(string script, bool wa
 
                 Assert.True(result.IsSuccess);
 
+                // Filter function can produce a 'WarnCheckPredicateUsage' warning. Any other warnings are unexpected.
+                var filteredErrors = result.Errors.Where(err => err.MessageKey != "WarnCheckPredicateUsage").ToList();

```

Comment:
>ilteredErrors = result.Errors.Where(err => err.MessageKey != "WarnCheckPredicateUsage").ToList(); [](http://example.com/codeflow?start=21&length=97)

From what I can see, the Filter expression in all [InlineData] rows access the scope (except the last one), so we shouldn't be getting those warnings here.